### ISSUE TITLE
Muse Hub Won't Load After First Time on Mac Big Sur / Downloads of Muse Sounds Immediately Lock[MU4 Issue]

### ISSUE THREAD

**Issue by VSTHero** [musescore/MuseScore#13929]
**Describe the bug**
After initial installation of Muse Hub on 13inch 2017 Mac Book Pro with Big Sur 11.6.1, Muse Hub will load for the first time, but once closed, it is unable to open again even after the computer is restarted.  Deleted all system files and then reinstalled and the same pattern happened again where initially it loaded on install but could not be reloaded after quitting the application.  Downloads of Muse Sounds also did not function during initial load (this may be separate issue).
 
**To Reproduce**
1. Install Muse Hub dmg
2. Double click Muse Hub from Applications
3. Enter Password to allow Helper to Install
3. Click to install any Muse Sound Library, download will lock on unknown length and no file/data will appear on drive (attempted to save on external SSD).
4. Quit Muse Hub
5. Click to reopen Muse Hub in Applications
6. Requests again to enter password to allow helper to install
7. Enter password.
8. Receive message that cannot communicate with helper program

*Screenshots**
<img width="447" alt="Screen Shot 2022-10-23 at 1 06 12 PM" src="https://user-images.githubusercontent.com/102884322/197405525-1b2bcc06-4525-4cd6-93a9-d844c21ad262.png">

<img width="260" alt="Screen Shot 2022-10-23 at 1 06 48 PM" src="https://user-images.githubusercontent.com/102884322/197405541-c36af8c0-7fc4-4c2f-b280-08f03b6c82fc.png">

**Platform information**
 - MacOS Big Sur 11.6.1

**Additional context**
I haven't tested if this issue occurs if no attempt is made to download Muse Sounds, that may be a separate issue or related.  I was able to download Muse Score 4 initial time Muse Hub was loaded.

**Comment by VSTHero** [comment]
Hi, so I seem to have resolved the conflict by chance; when I  reloaded Muse Hub I still had the mounted installer on my desktop.  When I removed it Muse Hub was able to load and the downloads for Muse Sounds are now functioning.

**Comment by Tantacrul** [comment]
If the issue is no longer persisting, I think I'll close. However, if you could possibly take a video where you replicate this behaviour, it would be SUPER useful. Thanks!

**Comment by escudeiro-daniel** [comment]
I have the same message and the same problem: "Opening and closing Muse Hub is very buggy, all of the icons are grey, I can't click on any of the icons, and it still says I'm downloading Muse Strings at zero KB per second." I just upload again and nothing happing. I used it on a macbook pro 8,1 with open-core patcher.

**Comment by mariuslacatus** [comment]
I have the same problem, I can't open Muse Hub. It is trying to install the helper everytime and it fails with the error:

Unable to connect to Helper. Please contact Support.
Unable to update helper settings: The operation couldn’t be completed. (com_muse_museservice.RemoteHelperError error 0.)

**Comment by stephaneboussuge** [comment]
Same for me, Museum start but stop immediately after one or two second...

**Comment by themodrnhakr** [comment]
This issue is very much NOT resolved...I was having issues with MuseHub, so I uninstalled a reinstalled. Now the Hub won't even open. Throws the "Unable to communicate with a helper application" error VSTHero was experiencing.

**Comment by JosephDanielss** [comment]
Mine says
"Muse Hub is having trouble connecting. Please check your internet connection an ensure any firewall you have is set correctly.". 

Pretty sure I have everything in order.

**Comment by cbjeukendrup** [comment]
The connection issue seems to be fixed. If other issues are still persisting, please report them separately at https://musehub.zendesk.com/.

**Comment by Rose22** [comment]
i'm still having this issue. i tried reinstalling muse hub and i tried rebooting, neither of them worked. i also tried to eject the disk image, but unlike the OP, that didn't fix it. i was excited check out the new musescore and the new playback engine, but due to this bug i can't.. this is a pretty critical bug

![image](https://user-images.githubusercontent.com/16426885/208163094-07143030-10ee-4b75-b5de-2b8244efab79.png)

**Comment by Synthmaniac** [comment]
> i'm still having this issue. i tried reinstalling muse hub and i tried rebooting, neither of them worked. i also tried to eject the disk image, but unlike the OP, that didn't fix it. i was excited check out the new musescore and the new playback engine, but due to this bug i can't.. this is a pretty critical bug
> 
> i'm on macos ventura 13.0.1, and like the OP i also tried to download the sounds onto an external harddrive, so that may be related?
> 
> ![image](https://user-images.githubusercontent.com/16426885/208163094-07143030-10ee-4b75-b5de-2b8244efab79.png)

I had the same issue, just solved it by disconnecting the hard drive, open up muse hub (worked), connecting the hard drive again and selected a library to install to the previously selected path on the hard drive. Seems to work flawlessly now.

**Comment by johngormley** [comment]
I had this problem too. However, as mentioned above, unplugging the hard drive solved the problem. MuseHub opened fine after that.

**Comment by skeller047** [comment]
Downloaded the MuseHub DMG.  Dragged to the Applications folder.   Started it.  It looked like it was downloading stuff, but I did not take a screenshot. 

The DMG file was still open on my screen; thinking I was finished with it I closed it.  The MuseHub app also closed.  I tried to reopen, but it would not run.   The app appears in my activity monitor - 
<img width="561" alt="Screenshot 2022-12-17 at 11 55 05 AM" src="https://user-images.githubusercontent.com/3360755/208263896-68003646-27bd-4fb3-a265-f5d10850e02c.png">

But I cannot launch MuseHub from the applications folder.  Trying to re-add it from the DMG, and selecting "replace", merely prompts me that it is already running.  My question - Where is MuseScore????
